lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -627,19 +627,30 @@ def test_data_variable_list(self):
         self.assertEqual(self.imos._data_variables[0].name, 'data_variable')
         self.assertEqual(self.imos._data_variables[1].name, 'random_data')
 
+    def test_check_data_variable_present(self):
+        self.imos.setup(self.good_dataset)
+        ret_val = self.imos.check_data_variable_present(self.good_dataset)
+        self.assertEqual(len(ret_val), 1)
```

Comment:
Yes, that's what it *should* return, but `ret_val` is a list, so if something's gone wrong in the code, it could have additional items. All tests are here to ensure that the checks return what they should.